package.json

```diff
@@ -6,6 +6,7 @@
     "url": "https://github.com/openfoodfoundation/openfoodnetwork"
   },
   "devDependencies": {
+    "jquery": "3.2.1",
```

Comment:
What is the problem that this is solving?

Reply:
I don't think this needs to be in either. I'll check.